src/idvc/dvc_interface.py

```diff
@@ -2894,7 +2896,8 @@ def CreatePointCloudPanel(self):
 2   300   750.2  209\n\
 etc.\n\
 Non-integer voxel locations are admitted, with reference volume interpolation used as needed.\n\
-The first point is significant, as it is used as a global starting point and reference for the rigid_trans variable.")
+The first point is significant, as it is used as a global starting point and reference for the rigid_trans variable.\n\
```

Comment:
I do not think this is true. If I remember correctly, the first point in the cloud is the closest to point 0.

Reply:
I have edited the documentation too. Please see https://github.com/TomographicImaging/iDVC/pull/264/files

Reply:
Where is this information recorded @paskino?

Reply:
in my head.

You are right. Forget this comment